subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -177,16 +177,22 @@ config BT_FAST_PAIR_BOND_MANAGER
 	select EXPERIMENTAL
 	select BT_FAST_PAIR_STORAGE_AK_BOND
 	help
-	  If this option is enabled, the Fast Pair subsystem tracks the Bluetooth bonds created
-	  through the Fast Pair Procedure and unpairs them if the Fast Pair Procedure is incomplete
-	  or the Account Key associated with the bonds is removed. It also unpairs the Fast Pair
-	  Bluetooth bonds on Fast Pair factory reset. Keep in mind that when the Fast Pair subsystem
-	  is disabled, the Bluetooth bonds are not tracked. Do not perform operations on the Fast
-	  Pair Bluetooth bonds apart from the Fast Pair factory reset when the Fast Pair subsystem
-	  is disabled. When this option is enabled, non-volatile storage is used to store the bond
-	  information. You can enable this option during DFU, but you cannot disable it during DFU
-	  if you used it earlier unless you erase whole Fast Pair non-volatile storage. Otherwise,
-	  the Fast Pair enable operation will fail.
+	  When this functionality is enabled, the Fast Pair subsystem tracks the Bluetooth bonds
+	  created through the Fast Pair Procedure and unpairs them if the procedure is incomplete or
+	  the Account Key associated with the bonds is removed. It also unpairs the Fast Pair
+	  Bluetooth bonds on Fast Pair factory reset, because the factory reset removes all Account
+	  Keys stored on device. When the Fast Pair subsystem is disabled, the Bluetooth bonds are
+	  not tracked. Do not perform operations on Fast Pair Bluetooth bonds, except for the Fast
+	  Pair factory reset when the Fast Pair subsystem is disabled. Additionally, do not perform
+	  Bluetooth bonding when the Fast Pair subsystem is disabled and bond management
+	  functionality is enabled, as this could cause the subsystem to incorrectly identify new
+	  bonds as Fast Pair bonds. To conclude, the easiest and safest option is to keep the Fast
+	  Pair enabled while using Bluetooth to ensure proper behavior. With this functionality, the
+	  non-volatile storage is used to store the bond information. You can enable the
```

Comment:
Can we make the DFU part as a separate section to improve readability? Same for part starting with `When the Fast Pair subsystem is disabled...`